### ISSUE TITLE
invalid token dead loop

### ISSUE THREAD

**Issue by lyhistory** [AlphaWallet/alpha-wallet-android#854]
app version: v2.10 (latest version from google play store)
phone model: Huawei P20 Pro
android version: 9
I have tried to Cancel and back arrow on the top left, still cannot break out from the dead loop
![deadlooop](https://user-images.githubusercontent.com/1522513/64712027-25525080-d4ed-11e9-9d62-dc1fabb917f8.gif)

**Comment by ronakdave195** [comment]
@JamesSmartCell May I request you to provide test steps and more description here so that we can try to reproduce this at our end to suggest any fixes.

**Comment by James-Sangalli** [comment]
@ronakdave195 try this one out: https://aw.app/AgAAAABeyqWsidFCvvhgVkaIHGjc1Iza8X_ll9wyMDI0MDcwNjIxMDAwMCswMzAwBQNVU0EAQVUBAQAJAGrEIIaM4BiSHG8w3PoSbkxQK4wVZJT6OsYjijJ7m2m_LraV7f0wW9lpRWcULiiUnl38ZBQqztkX5z6tJV-stIgc

**Comment by James-Sangalli** [comment]
@ronakdave195 it's possible this isn't an issue anymore but I don't know where it was solved because it wasn't tagged in any PR

**Comment by ronakdave195** [comment]
> @ronakdave195 try this one out: https://aw.app/AgAAAABeyqWsidFCvvhgVkaIHGjc1Iza8X_ll9wyMDI0MDcwNjIxMDAwMCswMzAwBQNVU0EAQVUBAQAJAGrEIIaM4BiSHG8w3PoSbkxQK4wVZJT6OsYjijJ7m2m_LraV7f0wW9lpRWcULiiUnl38ZBQqztkX5z6tJV-stIgc
> 
> It allows you to import a starwars token via a link

Thanks @James-Sangalli, I was able to Import a token with the link given above, however, have few feedbacks on the same:
1. Instead of Import-Cancel button, I was able to view Spawn-Cancel button
2. On trying to import a token 2nd time from the above link, I came across a Transaction Error message
3. On successful completion of importing a token, I wasn't able to view the Token on the Wallet tab/screen of the app. Is there a separate screen to view the imported tokens?

Kindly refer to the attached screenshots supporting my above comments. 
![import-token-transaction-error](https://user-images.githubusercontent.com/54934848/73719578-e6e05b00-4745-11ea-9b93-66fb0c3c2fa2.jpg)
![spawn-cancel-button](https://user-images.githubusercontent.com/54934848/73719579-e6e05b00-4745-11ea-87dd-6f0e37140006.jpg)
![no-token-show-wallet-screen](https://user-images.githubusercontent.com/54934848/73719580-e6e05b00-4745-11ea-9770-8d0c3e57402d.jpg)
